precompiles/staking/events.go

```diff
@@ -311,3 +306,21 @@ func (p Precompile) createStakingTxTopics(topicsLen uint64, event abi.Event, del
 
 	return topics, nil
 }
+
+// createValidatorTxTopics creates the topics for staking transaction CreateValidator.
```

Comment:
nit: The `CreateValidator` method needs to removed in the comment of the `createStakingTxTopics` method here: https://github.com/evmos/evmos/pull/2122/files#diff-45c7948de124db37615f89611485846d75ce8e3d3d66b88e74cc7b925c4b29afR287

Reply:
Yes, I forgot to check. has already been updated.